### PR TITLE
feat(person): Added person feature - OEL-1268

### PR DESCRIPTION
**Pull request by tibi2303**

### CODE REVIEW

File: src/compositions/bcl-person/.npmignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+__snapshots__
```
Comment (planctus) [inline src/compositions/bcl-person/.npmignore:1]: it's private, no need for the npmignore

File: src/compositions/bcl-person/data/data--listing.js [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+          {
+            label: "Clear",
+            type: "submit",
+            variant: "light",
+          },
+        ],
+      },
+    },
+  },
+  listing: {
```
Comment (planctus) [inline src/compositions/bcl-person/data/data--listing.js:48]: do we need everytime this huge file of data? Maybe we can dedicate some time to reorganise these data files we use "repeatedly" in many different "features", not in this pr, eh.. :)

Comment (tibi2303) [reply src/compositions/bcl-person/data/data--listing.js:48]: I agree, I think in the files for each dedicated feature package should be only the custom data for it, in rest we should "built"
it inside the story, I will take some time today to do that.

File: src/compositions/bcl-person/project.test.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  test("listing renders correctly", () => {
+    expect.assertions(1);
+    return expect(renderListing(demoListing)).resolves.toMatchSnapshot();
+  });
+
+  test("details renders correctly", () => {
+    expect.assertions(1);
+    return expect(renderDetails(demoDetails)).resolves.toMatchSnapshot();
+  });
+});
```
Comment (planctus) [inline src/compositions/bcl-person/project.test.js:47]: it seems you didn't add the package as a dependency for the bcl-twig-templates package.

File: src/compositions/bcl-person/data/data--listing.js (hunk 2 above)
Comment (tibi2303) [reply src/compositions/bcl-person/data/data--listing.js:48]: I think some packages we won't even need a custom data file for it.

File: src/compositions/bcl-person/.npmignore (hunk 1 above)
Comment (tibi2303) [reply src/compositions/bcl-person/.npmignore]: done.

File: src/compositions/bcl-person/project.test.js (hunk 3 above)
Comment (tibi2303) [reply src/compositions/bcl-person/project.test.js:47]: done.

**Comment by planctus** [comment]
can you check joinup, there is something in the left sidebar...
image

**Comment by tibi2303** [comment]
@planctus I can't see it..
image

**Comment by planctus** [comment]
this alignment seems approximative also in the default theme, but joinup is more visible

image

Also the border in the design seems closer to the heading than it is in our demo

**Review by planctus — APPROVED** [review]

**Pull request merged**